src/main/java/ro/msg/learning/shop/utils/TestingUtils.java

```diff
@@ -0,0 +1,224 @@
+package ro.msg.learning.shop.utils;
+
+import ro.msg.learning.shop.entitites.*;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class TestingUtils {
+	private static final UUID customerId = UUID.fromString("631e4cdd-bb78-4769-a0c7-cb948a9f1231");
+	private static final UUID location1Id = UUID.fromString("431e4cdd-bb78-4769-a0c7-cb948a9f1231");
```

Comment:
Please do a refactoring on the naming of the objects of this Util class. The name should reflect what the object represent. If I see the object name only in a test and I don't see the value of it, I should know what it contains. Also, you should use realistic object for testing purposes (a realistic product with a realistic name, category and so on). So please revisit the entire class and modify the objects to have realistic data inside.

Reply:
Refactored.